event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/feed/common/Page.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.services.eventsourcing.source.api.feed.common;
+
+import java.util.List;
+
+public class Page<T> {
```

Comment:
I think this is ok, but only if it is standalone and not linked to lots of other generics.

Reply:
Leaving it as such as this is generic container for links and data